nemo/collections/asr/models/aed_multitask_models.py

```diff
@@ -957,9 +932,14 @@ def _setup_transcribe_dataloader(self, config: Dict) -> 'torch.utils.data.DataLo
         Returns:
             A pytorch DataLoader for the given audio file(s).
         """
-        batch_size = min(config['batch_size'], len(config['paths2audio_files']))
+        if 'manifest_filepath' in config:
+            manifest_filepath = config['manifest_filepath']
+            batch_size = config['batch_size']
+        else:
+            manifest_filepath = os.path.join(config['temp_dir'], 'manifest.json')
```

Comment:
When is this else case occuring? Can you add a comment ?

Reply:
This is from the Transcription mixin where temp_dir was created to create a temp manifest when audio files are passed through list to .transcribe()